RELEASE.md

```diff
@@ -0,0 +1,9 @@
+## Release Finalization
+
+To cut a release:
+- Create a release branch in the format `release/0.99.0`
+  - New commits to this branch will trigger the `build` workflow and produce a lifecycle image: `buildpacksio/lifecycle:<commit sha>`
+- When ready to cut the release, manually trigger the `draft-release` workflow: Actions -> draft-release -> Run workflow -> Use workflow from branch: `release/0.99.0`
+  - This will create a draft release on GitHub using the artifacts from the `build` workflow run for the latest commit on the release branch
+- When ready to publish the release, edit the release page and click "Publish release"
+  - This will trigger the `post-release` workflow that will re-tag the lifecycle image from `buildpacksio/lifecycle:<commit sha>` to `buildpacksio/lifecycle:0.99.0`
```

Comment:
```suggestion
1. When ready to publish the release, edit the release page and click "Publish release". This will trigger the `post-release` workflow that will re-tag the lifecycle image from `buildpacksio/lifecycle:<commit sha>` to `buildpacksio/lifecycle:0.99.0`
```